### PR TITLE
Remove CodeEditorUtil

### PR DESCRIPTION
**Pull request by ivy-lli**
The browser test is not really a beauty, but as this functionality is only tested here at the moment, I didn't invest into move this code into an own pageobject.

### CODE REVIEW

File: integrations/standalone/tests/pageobjects/Table.ts [hunk 1]
```diff
@@ -115,8 +115,10 @@ export class Cell {
   async edit(value: string) {
     const expression = this.locator.getByRole('textbox');
     await expect(expression).toHaveAttribute('aria-expanded', 'false');
-    await expression.click();
-    await CodeEditorUtil.type(this.page, value);
+    const code = new CodeEditor(this.page, expression, expression, this.locator);
+    await code.focus();
+    await code.clearContent();
+    await this.page.keyboard.type(value);
```
Comment (weissreto) [inline integrations/standalone/tests/pageobjects/Table.ts:121]: code.fill(value) does something like this already

File: integrations/standalone/tests/mock/browser.spec.ts [hunk 2]
```diff
@@ -23,36 +23,45 @@ test.describe('Script browser', () => {
...
 
+    const category = task.macroInput('Category');
     await assertCodeHidden(page);
-    await page.getByLabel('Category').click();
+    await category.focus();
     await assertCodeVisible(page);
 
-    await CodeEditorUtil.fill(page, 'test 123 zag');
-    await page.getByRole('code').dblclick();
+    await page.keyboard.type('test 123 zag');
```
Comment (weissreto) [inline integrations/standalone/tests/mock/browser.spec.ts:31]: category.fill

**Review by weissreto — CHANGES_REQUESTED** [review]
I would try to hide thinks like page.keyboard.type also into the CodeEditor

File: integrations/standalone/tests/pageobjects/Table.ts (hunk 1 above)
Comment (ivy-lli) [reply integrations/standalone/tests/pageobjects/Table.ts:121]: fill does make a blur at the end

File: integrations/standalone/tests/mock/browser.spec.ts (hunk 2 above)
Comment (ivy-lli) [reply integrations/standalone/tests/mock/browser.spec.ts:31]: fill does make a focus, clear, type and blur. I only want to write...

File: integrations/standalone/tests/pageobjects/Table.ts (hunk 1 above)
Comment (ivy-lli) [reply integrations/standalone/tests/pageobjects/Table.ts:121]: I will make an own impl of the code editor for ScriptCell.

**Review by weissreto — APPROVED** [review]
🙏

**Pull request merged**